Storage/Rust/RustFirefoxSuggestion.swift

```diff
@@ -32,6 +40,8 @@ public struct RustFirefoxSuggestion {
             self.isSponsored = true
             self.iconImage = iconBytes.flatMap { UIImage(data: Data($0)) }
             self.fullKeyword = fullKeyword
+            self.clickInfo = .amp(blockId: blockId, advertiser: advertiser, iabCategory: iabCategory, reportingURL: URL(string: clickUrlString) ?? nil )
```

Comment:
```suggestion
            self.clickInfo = .amp(blockId: blockId, advertiser: advertiser, iabCategory: iabCategory, reportingURL: URL(string: clickUrlString))
```

`URL(string:)` can return `nil` already, so `?? nil` is redundant 😊